pkg/logql/fuzz.go

```diff
@@ -0,0 +1,11 @@
+// +build gofuzz
+
+package logql
+
+func FuzzParseExpr(data []byte) int {
+	_, err := ParseExpr(string(data))
+	if err != nil {
+		return 0
```

Comment:
what does 0 or 1 means for the fuzzer ?

Reply:
From [the documentation](https://github.com/dvyukov/go-fuzz#usage): 

> Data is a random input generated by go-fuzz, note that in most cases it is invalid. The function must return 1 if the fuzzer should increase priority of the given input during subsequent fuzzing (for example, the input is lexically correct and was parsed successfully); -1 if the input must not be added to corpus even if gives new coverage; and 0 otherwise; other values are reserved for future use.

Reply:
Thanks @AdamKorcz